app/2.0/docs/devguide/encapsulated-style.md

```diff
@@ -0,0 +1,639 @@
+	  <p class="myclass">So am I.</p>
+  </template>
+</dom-module>
+<x-foo></x-foo>
+<!-- The following paragraph uses the document-level stylesheet. -->
+<p class="myclass">I have nothing to do with x-foo. Because of encapsulation, x-foo's styles won't leak to me.</p>
+```
+
+For a detailed explanation of shadowDOM as it applies to Polymer, see [arthur's article]().
+
+For an exploration of the shadow DOM v1 API, see [Shadow DOM v1: Self-Contained Web Components](https://developers.google.com/web/fundamentals/getting-started/primers/shadowdom).
+
+### Using inheritance from document-level styles
+
+When used in an HTML document, your element will still inherit any styling information that applies to its parent element:
+
+[See it on Plunker](http://plnkr.co/edit/cHSjdQTa0h6fWXAygkje?p=preview)
+
+```html
+<!-- Document-level stylesheet -->
+<style>
+p {
+	font-family: sans-serif;
+	color:blue;
+}
+</style>
+</head>
+<dom-module id='x-foo'>
+  <template>
+    <div>
+      I inherit styles from x-foo's parent in the light DOM.
+      I'm also sans-serif and blue.
+    </div>
+  </template>
+</dom-module>
+
+<!-- This paragraph uses document-level styles: -->
+<p>I'm sans-serif and blue.</p>
+
+<!-- And the text within x-foo inherits style from the paragraph element: -->
+<p><x-foo></x-foo></p>
+```
+
+Styles declared inside shadow DOM will override styles declared outside of it:
+
+[See it on Plunker](http://plnkr.co/edit/0Fid1Gupd0jk9jggAKuv?p=preview)
+
+```html
+<!-- Document-level stylesheet -->
+<style>
+	p { 
+		font-family: sans-serif;
+		color:blue;
+    }
+</style>
+<dom-module id='x-foo'>
+  <template>
+    <!-- Encapsulated, element-level stylesheet -->
+    <style>
+		p {
+			font-family: sans-serif;
+			color:green;
+		}
+		</style>
+		<p>I'm green.</p>
+  </template>
+</dom-module>
+
+<p>I'm blue.</p>
+<p><x-foo></x-foo></p> 
+```
+
+### Styling the host element
+
+
+The element to which shadow DOM is attached is known as the host. To style the host, use the `:host` selector.
+
+Inheritable properties of the host element will inherit down the shadow tree, where they apply to the shadow children.
+
+[See it on Plunker](http://plnkr.co/edit/7771DvsQ3iPWnn2gEIf8?p=preview)
+
+```html
+<dom-module id="x-foo">
+	<template>
+		<style>
+		:host {
+			font-family: sans-serif;
+			color: green;
+			display: block;
+			border: 1px solid;
+		}	
+	</style>
+	<p>I'm green.</p>
+	<div>I'm green too.</div>
+	<span>We're all green...</span>
+	</template>
+</dom-module>
+<x-foo></x-foo>    
+```
+
+You can also style the host element from outside - for example, using a type selector:
+
+[See it on Plunker](http://plnkr.co/edit/AHXFX0zeQTbO2rGELTbS?p=preview)
+
+```html
+x-foo {
+	background-color: blue;
+}
+```
+
+#### Using selectors 
+
+You can use CSS selectors to determine when and how to style the host. In this code sample:
+
+* The rule `:host { font-family: sans-serif; }` always matches `<x-foo>` elements
```

Comment:
Rather than have the whole rule, I'd just include the selector. "The selector `:host` matches any `<x-foo>` element."